litt/src/main.rs

```diff
@@ -87,6 +104,332 @@ fn show_failed_documents_error(index: &Index) {
     }
 }
 
+fn read(history: &mut Vec<String>) -> Result<String, LittError> {
```

Comment:
Why don't we create the history vector internally instead of having it as a parameter to the function?
Do we need it after processing at the calling site?